cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -836,6 +839,158 @@
+class ShardMonitor:
+    def __init__(self, analyzer: ShardAnalyzer):
+        self.analyzer = analyzer
+        self.reference_shards: dict[str, ShardInfo]
+        self.latest_shards: list[ShardInfo]
+        self.seq_deltas: dict[str, int]
+        self.size_deltas: dict[str, float]
+
+        self.table_filter: str|None = None
+        self.sort_by: str = 'heat'
+
+    def monitor_shards(self, table_filter: str | None, interval_in_seconds: int = 5, repeat: int = 10, n_shards: int = 40, sort_by: str = 'heat'):
```

Ruff (0.12.2)

Line too long (146 > 120)

(E501)